.github/workflows/dispatch-with-files.yaml

```diff
@@ -0,0 +1,47 @@
+name: Dispatch with files
+
+on:
+  schedule:
+    - cron: '5 2 * * *'
+
+  workflow_dispatch:
```

Comment:
In addition to cron, we will have a button on the github actions interface to trigger manually the dispatch with files (and the normal dispatch)